packages/jest-preset-mc-app/package.json

```diff
@@ -51,6 +51,6 @@
   "peerDependencies": {
     "@testing-library/react": "9.x",
     "enzyme": "3.x",
-    "jest": "23.x"
+    "jest": "23.x || 24.x"
```

Comment:
This is odd. We're using jest@24 for months now. Should we change this to `24.x` only instead?

Reply:
Yeah, might be a good idea.